packages/integration-tests/tests/retrospective-demo/smoke.test.ts

```diff
@@ -0,0 +1,40 @@
+import config from '../config'
+import {test, expect} from '@playwright/test'
+
+test.describe('retrospective-demo / smoke test', () => {
+  test('displays a welcome dialog', async ({page}) => {
+    await config.goto(page, '/retrospective-demo')
+    await expect(page.locator('button', {hasText: 'Start Demo'})).toBeVisible()
+  })
+
+  test('displays the reflect page first', async ({page}) => {
+    await config.goto(page, '/retrospective-demo')
+    await page.click('text=Start Demo')
+    await expect(page.locator('h1:has-text("Reflect")')).toBeVisible()
+    await expect(page.locator('[data-cy=reflection-column-Start]')).toBeVisible()
+    await expect(page.locator('[data-cy=reflection-column-Stop]')).toBeVisible()
+    await expect(page.locator('[data-cy=reflection-column-Continue]')).toBeVisible()
+  })
+
+  test('navigates to the group phase when clicking the next button', async ({page}) => {
+    // Given a new visitor to the retrospective demo page
+    await config.goto(page, '/retrospective-demo')
+
+    // When they start the demo
+    await page.click('text=Start Demo')
+
+    // When they enter a response
+    const startTextbox = '[data-cy=reflection-column-Start] [role=textbox]'
+    await page.click(startTextbox)
+    await page.type(startTextbox, 'Start doing this')
+    await page.press(startTextbox, 'Enter')
+
+    // When they click the "Next" button twice
+    await page.click('[data-cy="next-phase"]:not([disabled=true])')
+    await page.click('[data-cy="next-phase"]:not([disabled=true])')
```

Comment:
I'm leveraging some of the leftover `data-cy` HTML attributes as part of this PR. However, it's increasingly popular to use integration tests as a way of improving the accessibility (a11y) of your frontends.

Frameworks like `@testing-library`, encourage finding elements by text, role, or HTML attribute type instead of these data attributes, which screen-readers ignore. For example, this "clickable div" is harder for a screen reader to understand vs. using the `button` HTML attribute (or `role=button`). It's also challenging to identify in this test, which is why we probably added the data attribute.

See [`@testing-library`'s docs](https://testing-library.com/docs/queries/about) and [this blog post](https://developers.livechat.com/updates/livechat-automated-a11y-testing) for more examples.

This is more of a long-term consideration, but another added benefit we might get from adding back an integration/through-the-glass testing setup.

Reply:
I'm sure @enriquesanchez will love this!

Reply:
i'm 100% in favor of this. i propose we get rid of all the data-cy fields & if we can't get to a field via attribute then our a11y needs improvement!

Reply:
Great! I've got another branch (https://github.com/ParabolInc/parabol/compare/feature/continue-testing-demo?expand=1) that I've got some additional tests. I'll start following this pattern moving forward.